src/tests/system/tests/test_ipa.py

```diff
@@ -122,3 +124,70 @@ def test_ipa__hostpublickeys_by_ip(client: Client, ipa: IPA, public_keys: list[s
     assert len(public_keys) == len(result.stdout_lines), "Did not get expected number of public keys!"
     for key in public_keys:
         assert f"{ip} {key}" in result.stdout_lines, "Did not get expected public keys!"
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__check_gssapi_authentication_indicator(client: Client, ipa: IPA):
+    """
+    :title: Verify GSSAPI authentication indicator
+    :description:
+        Checks that the kerberos ticket was pre-authentication using GSSAPI using authentication indicators
+    :setup:
+        1. Configure SSSD for sudo and gssapi
+        2. Start SSSD
+        3. Create sudo configuration that allows user to run SUDO rules
+    :steps:
+        1. Switch to ipa user
+        2. Obtain Kerberos ticket.
+        3. Try 'sudo -l' as user
+        4. Check if acquired service ticket has req. indicators: 0 in sssd_pam.log
+        5. Change config for pam section with pam_gssapi_services and pam_gssapi_indicators_map
```

Comment:
Update step 5 and add the restart

Update config with 'pam_gssapi_service, pam_gssaspi_indicator_map and restart sssd

:expectedresult:

Configuration is updated and SSSD is restarted